packages/typespec-test/test/anomalyDetector/generated/typespec-ts/src/anomalyDetectorClient.ts

```diff
@@ -51,5 +51,7 @@ export default function createClient(
     options
   ) as AnomalyDetectorClient;
 
+  client.pipeline.removePolicy({ name: "ApiVersionPolicy" });
```

Comment:
there's is a path level api version, we should not add this policy as it's for query api version parameter.